src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/StringModule.fs

```diff
@@ -41,8 +41,8 @@ type StringModule() =
         let e6 = String.concat "foo" ["bar"]
         Assert.AreEqual("bar", e6)
         
-        let e7 = String.concat "foo" ["bav";"baz"]
-        Assert.AreEqual("bavfoobaz", e7)
+        let e7 = String.concat "foo" ["bar";"baz"]
+        Assert.AreEqual("barfoobaz", e7)
```

Comment:
We can't change this back, as it introduces the string "barf" to the codebase. :laughing:   We had to change this prior to open sourcing on Codeplex because internal code scrub tools complained. Would recommend open edition repo accepts our version.